cli/src/main/kotlin/commands/NotifierCommand.kt

```diff
@@ -52,12 +54,20 @@ class NotifierCommand : CliktCommand(name = "notify", help = "Create notificatio
         .convert { it.absoluteFile.normalize() }
         .inputGroup()
 
+    private val labels by option(
+        "--label", "-l",
+        help = "Set a label in the ORT result, overwriting any existing label of the same name. This label can be " +
```

Comment:
Nit: Should the help text explicitly state that the label is only transient and will not be persisted? The current statement sounds as if the result would be modified. Maybe something like "Temporarily update a label in the ORT result before it is passed to the notifier script, overwriting any existing label of the same name. This label can then be accessed by the notifier script, but the modified ORT result is not persisted. ..."

Reply:
At first I also though such a hint would make sense, but then again, where should the label eventually be persisted to, as no output result is written anyway? And *if* an output would be written, it would also be persisted.

The current text matches the one in the other CLI tools except for the inserted "This label can be accessed in the used notifier script" sentence. As a compromise, how about:

> Set a label in the ORT result passed to the notifier script, overwriting any existing label of the same name. Can be used multiple times. For example: --label distribution=external